tabbycat/participants/models.py

```diff
@@ -332,6 +332,26 @@ class Meta:
 
     def __str__(self):
         return str(self.name)
+    
+    def get_speaker_points(self, s):
+        try:
+            return int(s.score)
```

Comment:
This is not necessarily an integer; the scoring could be decimal or the average of many AdjScores could use them.